articles/key-vault/key-vault-ovw-storage-keys.md

```diff
@@ -162,20 +157,19 @@ Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -Accoun
 
 ### Regeneration
 
-Setting the regeneration period using the following commands.
+Setting the regeneration period to 3 days using the following commands.
 
 ```powershell
-$regenPeriod = [System.Timespan]::FromDays(3)
-Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -AccountResourceId /subscriptions/subscriptionId/resourceGroups/yourresgroup1/providers/Microsoft.Storage/storageAccounts/yourtest1 -ActiveKeyName key2 -RegenerationPeriod $regenPeriod
+Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -AccountResourceId /subscriptions/subscriptionId/resourceGroups/yourresgroup1/providers/Microsoft.Storage/storageAccounts/yourtest1 -ActiveKeyName key2 -RegenerationPeriod "03.00:00:00"
```

Comment:
@BrucePerlerMS I prefer it the original way here as well Bruce I think it is more programmatic. I like to minimize hard coding values and long strings

Reply:
Agreed. I'll reject the PR.